detail/namespace.vue

```diff
@@ -172,30 +230,57 @@ export default {
       }
 
       return null;
-    }
+    },
   }
 };
 </script>
 
 <template>
-  <div>
+  <Loading v-if="$fetchState.pending" />
+  <div v-else>
     <div class="mb-20">
       <h3>{{ t('namespace.resources') }}</h3>
-      <ResourcesSummary state-key="namespace.resourceStates" :value="accumulatedStateCounts" :required-states="['success', 'info', 'warning', 'error', 'unknown']" />
+      <ResourcesSummary
+        state-key="namespace.resourceStates"
+        :value="accumulatedStateCounts"
+        :required-states="summaryStates"
+      />
     </div>
     <ResourceTabs v-model="value" :mode="mode">
       <Tab :name="t('namespace.resources')">
-        <SortableTable default-sort-by="error" :table-actions="false" :row-actions="false" :rows="namespacedResourceCounts" :headers="headers">
+        <SortableTable
+          default-sort-by="error"
+          :table-actions="false"
+          :row-actions="false"
+          :rows="namespacedResourceCounts"
+          :headers="headers"
+        >
           <template #col:type="{row}">
             <td>
-              <n-link v-if="typeListLocation(row.schema)" :to="typeListLocation(row.schema)">
+              <n-link
+                v-if="typeListLocation(row.schema)"
+                :to="typeListLocation(row.schema)"
+              >
                 {{ row.schema.pluralName }}
               </n-link>
               <span v-else>{{ row.schema.pluralName }}</span>
             </td>
           </template>
         </SortableTable>
       </Tab>
+
+      <Tab :name="t('namespace.workloads')">
+        <ResourceTable
+          name="workloads"
+          :table-actions="false"
+          :row-actions="false"
```

Comment:
Is there a particular reason we can't allow row actions here? If this is disabled because the person viewing the page may not have permission to alter workloads the 'health' column probably needs to be changed too, since it allows the user to scale workloads. If the user viewing this page can edit workloads then we should give them row actions.

Reply:
I did not put just because it was not requested. Enabling now.

Reply:
To be fair, it would be also great to have a button to create the items from the table too, as for instance it's not so easy to get, plus it can be faster too.